Frontend-Development/Contribute/Readme.md

```diff
@@ -75,7 +75,6 @@
 - [The Center for Connected Learning and Computer-Based Modeling](https://summerofcode.withgoogle.com/archive/2018/organizations/5109641792978944)
 - [The Palisadoes Foundation](https://summerofcode.withgoogle.com/archive/2021/organizations/4835102856577024)
 - [Webiu](https://github.com/scorelab/Webiu)
-- [Wikimedia Foundation](https://github.com/wikimedia/portals/tree/master/docs)
```

Comment:
The links are different, and The Wikimedia Foundation is not stated twice.

After the main title, you can modify it and insert the specific line title in brackets.

example:
Wikimedia (Github - Portal  Doc) 
Wikimedia (GSOC).

Reply:
Done